website/docs/guides/using-terraform-templates.mdx

```diff
@@ -12,7 +12,7 @@ This guide will show you how to use a template to create a Terraform resource in
 ## CLI guide
 
 ### Pre-requisites
-- Install [Weave GitOps Enterprise](installation.mdx#weave-gitops-enterprise) with [TF-Controller installed](installation.mdx#tf-controller) and [TLS enabled](../configuration/tls.md).
+- Install [Weave GitOps Enterprise](installation.mdx#weave-gitops-enterprise) with [TF-Controller installed](installation.mdx#installing-the-tf-controller) and [TLS enabled](../configuration/tls.md).
```

Comment:
the link should be 
`[TF-Controller installed](installation.mdx/#installing-the-tf-controller)` i think